src/ILLink.Shared/TrimAnalysis/RequireDynamicallyAccessedMembersAction.cs

```diff
@@ -26,6 +26,9 @@ public void Invoke (in MultiValue value, ValueWithDynamicallyAccessedMembers tar
 					&& genericParam.HasDefaultConstructorConstraint ()) {
 					// We allow a new() constraint on a generic parameter to satisfy DynamicallyAccessedMemberTypes.PublicParameterlessConstructor
 				} else if (uniqueValue is ValueWithDynamicallyAccessedMembers valueWithDynamicallyAccessedMembers) {
+					if (uniqueValue is NullableValueWithDynamicallyAccessedMembers nullableValue) {
+						MarkTypeForDynamicallyAccessedMembers (nullableValue.NullableType, nullableValue.DynamicallyAccessedMemberTypes);
```

Comment:
I think this is missing the marking of `Nullable<T>` itself.

Reply:
`nullableValue.NullableType` represents the `Nullable<>`, and since `NullableValueWithDynamicallyAccessedMembers` has an unknown underlying type, we wouldn't want to mark anything else, would we?

Reply:
Sorry - I'm blind. Please ignore.